.github/workflows/remove_review.yaml

```diff
@@ -9,46 +9,25 @@ on:
       - '*'
 
 jobs:
-  check_ns:
-    name: "Check namespace exist"
-    runs-on: self-hosted
-    outputs:
-      ns_exist: ${{ steps.check_ns.outputs.exist }}
-    steps:
-      - name: Set env for namespace
-        run: |
-          REF_NAME=${{ github.event.ref }}
-          echo "REVIEW_NAMESPACE=$(echo review-${REF_NAME##*/} | sed 's/\_/-/g')" >> $GITHUB_ENV
-      - name: "Check namespace exist"
-        id: check_ns
-        run: |
-          kubectl get ns ${{ env.REVIEW_NAMESPACE }} > /dev/null 2>&1 && \
-          echo "::set-output name=exist::true" || \
-          echo "::set-output name=exist::false"
-  converge:
+  remove_review:
     name: "Remove review"
-    needs: [check_ns]
-    if: needs.check_ns.outputs.ns_exist == 'true'
     runs-on: self-hosted
     env:
-      WERF_SECRET_KEY: ${{ secrets.WERF_SECRET_KEY }}
-      PULL_USER: ${{ secrets.PULL_USER }}
-      PULL_PERSONAL_ACCESS_KEY: ${{ secrets.PULL_PERSONAL_ACCESS_KEY }}
+      KUBE_CONFIG_BASE64_DATA: ${{ secrets.TEST_CLUSTER_KUBE_CONFIG_BASE64_DATA }}
```

Comment:
test?

Reply:
I'm not a creator of that secret, so I can't edit or rename it...